docs/demos/telecom-customer-churn-airflow/telecom-customer-churn-airflow.md

```diff
@@ -0,0 +1,302 @@
+- Have [Airflow](/tools-and-applications/airflow/airflow/) running in a cluster and point Airflow to the cloned git repository.
+- Have access to some S3 storage (this guide uses ODF with a bucket created in the namespace "airflow").
+- Have [Red Hat OpenShift Data Science](/getting-started/openshift-data-science/) (RHODS) running in a cluster. Make sure you have admin access in RHODS, or know someone who does.
+
+!!! note
+    Note: You can use [Open Data Hub](/getting-started/opendatahub/) instead of RHODS, but some instructions and screenshots may not apply
+
+### 1: Open up Airflow
+
+You find the route to the Airflow console through this command: `oc get route -n airflow`
+
+![Airflow Route](img/Airflow_Route.png)
+
+Enter it in the browser and you will see something like this:
+
+![Airflow Main Hub](img/Airflow_main_hub.png)
+
+Keep that open in a tab as we will come back to Airflow later on.
+
+### 2: Add Elyra as a Custom Notebook Image
+
+It's possible to build pipelines by creating an Airflow DAG script in python. Another, arguably simpler, method is to use Elyra to visually build out the pipeline and then submit it to Airflow.  
+Most of this demo is going to be revolving around using Elyra together with Airflow, but at the very end, there will be a bonus section for how to use Airflow independently.
+
+To get access to Elyra, we will simply import it as a custom notebook image.  
+Start by opening up RHODS by clicking on the 9-square symbol in the top menu and choosing "Red Hat OpenShift Data Science".
+
+![Open RHODS](img/Open_RHODS.png)
+
+Then go to Settings -> Notebook Images and press "Import new image".  
+If you can't see Settings then you are lacking sufficient access. Ask your admin to add this image instead.
+
+![Import Notebook Image](img/Import_Notebook_Image.png)
+
+Under Repository enter: `quay.io/eformat/elyra-base:0.2.1` and then name it something like `Elyra`.
+
+### 3: Create a RHODS workbench
+
+A workbench in RHODS lets us spin up and down notebooks as needed and bundle them under Projects, which is a great way to get easy access to compute resources and keep track of your work.  
+Start by creating a new Data Science project (see image). I'm calling my project 'Telecom Customer Churn', feel free to call yours something different but be aware that some things further down in the demo may change.
+
+![Create Data Science Project](img/Create_data_science_project.png)
+
+After the project has been created, create a workbench where we can run Jupyter.
+There are a few important settings here that we need to set:
+
+- **Name:** Customer Churn
+- **Notebook Image:** Elyra
+- **Deployment Size:** Small
+- **Environment Variables:** Secret -> AWS with your AWS details
+
+![Workbench Settings](img/Workbench_Settings.png)
+
```

Comment:
Sure you need 20GB of storage? I would lower this, otherwise people will replicate your demo over and over, consuming storage for nothing.

Reply:
I do not 😅 
Updated it